.github/workflows/ci.yml

```diff
@@ -1,76 +1,121 @@
-name: ci
+name: Test
```

Comment:
I'm no expert on these github workflows now - @bretbrownjr any comments?  Specifically I'm wondering about non-Linux platforms.  Also, @steve-downey did this come from our template or do we need to update?

Reply:
@steve-downey , is there a way to run these flows on local environment? e.g. git API to span a Docker for every job, but on my local setup. I am just curious if I am doing a change, if I can run all jobs on my MAC OS / Ubuntu.

Reply:
I have not found a way to simulate any of this locally. I've ended up enabling actions on PRs and repeated force-pushes to a branch to trigger a run.

Reply:
I took a bunch of it from https://github.com/beman-project/example/pull/6. I also have some possible updates to deal with Node version warnings over here: https://github.com/steve-downey/lambda-fun/pull/2/files